Source/WebCore/platform/graphics/RenderingMode.h

```diff
@@ -34,7 +34,8 @@ enum class RenderingPurpose : uint8_t {
     LayerBacking,
     Snapshot,
     ShareableSnapshot,
-    MediaPainting
+    MediaPainting,
+    LocalBufferForSnapshot
```

Comment:
We should have some consistency between these options:
```
Snapshot,
ShareableSnapshot
LocalBufferForSnapshot
```

`LocalBufferForSnapshot` makes a Shareable (with the UI-process) buffer. So let's call the new value `ShareableLocalSnapshot`, and put it before `MediaPainting`.

Reply:
Fixed.